llama_index/callbacks/open_inference_callback.py

```diff
@@ -0,0 +1,315 @@
+"""
+Callback handler for retrieval-augmented generation data in OpenInference
```

Comment:
```suggestion
Callback handler for the generation data in OpenInference. 
```

Reply:
I would link the open-inference spec here

Reply:
Addressed in [f968ba9](https://github.com/jerryjliu/llama_index/pull/6998/commits/f968ba9d09d00cdb3b3662d37667b81c74ccb323)